crates/bevy_pbr/src/render_graph/lights_node.rs

```diff
@@ -86,7 +86,9 @@ pub fn lights_node_system(
     let state = &mut state;
     let render_resource_context = &**render_resource_context;
 
-    let ambient_light: [f32; 4] = ambient_light_resource.color.into();
+    // premultiply ambient brightness
+    let ambient_light: [f32; 4] =
+        (ambient_light_resource.color * ambient_light_resource.brightness).into();
```

Comment:
FYI, I've found it's best to convert the brightness from gamma to linear space before multiplying to get perceptually linear brightness control. There should be a method already in bevy to do that.